test/integration_tests/test_models.py

```diff
@@ -6,8 +6,9 @@
     XLMR_BASE_ENCODER,
     XLMR_LARGE_ENCODER,
 )
+from torchtext.utils import get_asset_local_path
 
-from ..common.assets import get_asset_path
+from ..common.assets import conditional_remove, get_asset_path
```

Comment:
Mixing fixtures/assets from unit tests with integration tests is a code smell for me.

Reply:
Removed and replaced with pytest fixtures as per your recommendation